docs/contributors/getting-started.md

```diff
@@ -7,80 +7,137 @@ Whenever you are stuck or unsure, please open an issue or
 [ask on Gitter](https://gitter.im/scalameta/metals). This project follows
 [Scalameta's contribution guidelines](https://github.com/scalameta/scalameta/blob/master/CONTRIBUTING.md).
 
+## Requirements
+
+You will need the following applications installed:
+
+- Java 8, make sure `JAVA_HOME` points to a Java 8 installation and not Java 11.
+- `git`
+- `sbt` (for building a local version of the server)
+
 ## Project structure
 
 - `metals` contains a Scala implementation of a language server based on
-  Scalameta
+  Scalameta.
 - `mtags` source file indexer for Java and Scala.
-- `vscode-extension` contains a Visual Studio Code extension, implementing a
-  client for the language server
-- `tests` directory for unit tests
-- `test-workspace` directory for manually testing the vscode plugin locally
+- `tests/input` example Scala code that is used as testing data for unit tests.
+- `tests/unit` moderately fast-running unit tests.
+- `tests/slow` slow integration tests.
+- `test-workspace` demo project for manually testing Metals through an editor.
 
 ## Related projects
```

Comment:
maybe add metals-vscode to the related projects?

Reply:
Good idea!